scripts/gen_keys.js

```diff
@@ -58,7 +58,7 @@ function addKeyProperties(key) {
 console.log('Generating keypair')
 
 cp.exec(
-  'openssl genrsa 2048 | ../node_modules/fxa-jwtool/node_modules/pem-jwk/bin/pem-jwk.js',
+  'openssl genrsa 2048 | ../node_modules/pem-jwk/bin/pem-jwk.js',
```

Comment:
I think this will break `npm 2` installations, npm 2 nests the modules , and npm3+ puts everything into a single directory

Reply:
I could add a fork here to check for which one exists, rather than hard-coding it.  Are we committed to npm2-compatibility for a while?